data/newMigrations/014-service_entry_provider.js

```diff
@@ -0,0 +1,22 @@
+exports.up = (knex) => {
+    return knex.schema.createTable('service_entry_providers', function (tbl) {
+        tbl.increments('service_entry_provider_id').primary();
```

Comment:
In this case, when using `.increments()`, knex will automatically set this as a primary key, so you don't need `.primary()`.